src/transaction.c

```diff
@@ -157,8 +157,10 @@ static bool is_valid_elements_tx(const struct wally_tx *tx)
 
 bool clone_bytes(unsigned char **dst, const unsigned char *src, size_t len)
 {
-    if (!len)
+    if (!len) {
+	*dst = NULL;
         return true;
```

Comment:
nit ([needed for CI](https://github.com/ElementsProject/libwally-core/blob/master/.gitlab-ci.yml#L14-L15)):
```suggestion
        *dst = NULL;
        return true;
```